scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -611,31 +598,27 @@ protected String prepareJobOutput(TaskFlowJob job, String localOutputFolder, Str
      * @param job
      * @param localInputFolder path to the input folder on local machine if null, no actions
      *                         will be performed concerning the input data for this job
-     * @param pushURL          the url where input data is to be pushed before the job
+     * @param pushURLs         the urls where input data is to be pushed before the job
      *                         submission
      * @param newFolderName    name of the new folder to be created
      * @return String representing the updated value of the push_url
      */
-    protected String prepareJobInput(Job job, String localInputFolder, String pushURL, String newFolderName)
+    protected String prepareJobInput(Job job, String localInputFolder, List<String> pushURLs, String newFolderName)
             throws NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)